### PR TITLE
Fix: Nord Theme: "Mark all" menu item

### LINKED ISSUE #4978: Feed menu: "mark as read" is to big
If you click manage in this menu:
image

is "Mark as read" supposed to be so large? That's with the latest edge, Nord theme.

_Originally posted by @Frenzie in https://github.com/FreshRSS/FreshRSS/issues/4977#issuecomment-1368481030_

### PR DESCRIPTION
**Pull request by math-GH**
Closes #4978

Changes proposed in this pull request:

- CSS for Nord theme
- (S)CSS for other themes. font-size: inherit makes the code a bit clearer

How to test the feature manually:

1. see #4978

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: p/themes/Alternative-Dark/adark.css [hunk 1]
```diff
@@ -348,7 +348,7 @@ a.btn {
 .dropdown-menu .item > .as-link {
 	padding: 0 22px;
 	line-height: 2.5;
-	font-size: 0.8rem;
+	font-size: inherit;
```
Comment (Frenzie) [inline p/themes/Alternative-Dark/adark.css:351]: Wouldn't removing the line have the same effect?

Comment (math-GH) [reply p/themes/Alternative-Dark/adark.css:351]: I thought so, too. But it does not have the same effect.

`font-size: inherit` = 12,8pixel
deleted line = 13.33 pixel

To be honest: I did not understand why it has not the same effect.

Comment (Alkarex) [reply p/themes/Alternative-Dark/adark.css:351]: > I did not understand why it has not the same effect

I have not checked, but this is typically due to the fact that without this line, another rule is applied from another selector.

Comment (Frenzie) [reply p/themes/Alternative-Dark/adark.css:351]: The inspector will say which rule. Anyway, just double checking.

**Pull request merged**